README.md

```diff
@@ -42,12 +42,12 @@ Repository at <https://github.com/cketti/ckChangeLog>.
 2. Create translations of this `changelog_master.xml` file in files named `changelog.xml` under
 language-specific versions of `res/xml/`, e.g. `res/xml-de/changelog.xml`.
 
-3. Display the change log dialog by putting the following code in your activity's `onCreate()` method:
+3. Display the change log dialog by putting the following code in your Activity's `onCreate()` method:
 
   ```java
-  ChangeLog cl = new ChangeLog(this);
-  if (cl.isFirstRun()) {
-      cl.getLogDialog().show();
+  DialogChangeLog changeLog = DialogChangeLog.newInstance(this);
```

Comment:
Thanks to the in-person clarification the other day I understand that you actually want to create a new instance every time. Forgot what I wrote above - certainly I confused things. _sigh_